extension/popup/popupController.jsx

```diff
@@ -50,6 +50,16 @@ this.popupController = (function() {
     const init = async () => {
       const userSettings = await settings.getSettings();
       if (!userSettings.collectTranscriptsOptinAnswered) {
+        console.log("Searching for open onboarding tabs...");
+        for (const tab of await browser.tabs.query({
+          url: ["moz-extension://*/onboarding/onboard.html"]
+        })) {
+          console.log("Found an open onboarding tab!");
+          window.close();
```

Comment:
This should happen at the end, it probably stops everything. In fact it might be necessary to do `setTimeout(window.close)` just to make sure other things have time to run before the popup closes.